packages/eslint-config-kibana/.eslintrc.js

```diff
@@ -12,7 +13,13 @@ module.exports = {
     'prefer-object-spread',
     'jsx-a11y',
   ],
-  
+
+  settings: {
+    react: {
+      version: REACT_VERSION_RANGE.replace(/[\D\.]/g, '').split('.').slice(0, 2).join('.'),
```

Comment:
__issue:__ this construct looks strange, it removes all non-digits **and dots** and then tries to split result by dots.... Btw, can we use 3rd-party dev deps inside of `.eslintrc.js` (e.g. `semver.coerce` from `semver`)?